src/Microsoft.AspNetCore.Mvc.TagHelpers/Cache/DistributedCacheTagHelperFormatter.cs

```diff
@@ -14,18 +14,18 @@ namespace Microsoft.AspNetCore.Mvc.TagHelpers.Cache
     /// </summary>
     public class DistributedCacheTagHelperFormatter : IDistributedCacheTagHelperFormatter
     {
-        
+
         /// <inheritdoc />
         public Task<byte[]> SerializeAsync(DistributedCacheTagHelperFormattingContext context)
         {
-            if (context == null) 
+            if (context == null)
             {
-                throw new ArgumentNullException(nameof(context));    
+                throw new ArgumentNullException(nameof(context));
             }
-            
-            if (context.Html == null) 
+
+            if (context.Html == null)
             {
-                throw new ArgumentNullException(nameof(context.Html));    
+                throw new ArgumentNullException(nameof(context.Html));
             }
```

Comment:
/cc @sebastienros I remember having the same comment when you wrote the code but there was a reason. What was the reason?